app/controllers/api/v1/monsters_controller.rb

```diff
@@ -0,0 +1,14 @@
+class Api::V1::MonstersController < ApplicationController
+  protect_from_forgery unless: -> { request.format.json? }
+
+  def index 
+    render json: {monsters: Monster.all}
+  end
+
+  def update 
+    @monster = Monster.find(params[:id])
+    @monster.votes = params[:votes]
+    @monster.save
+    render json: {monster: @monster}
+  end
```

Comment:
I see here this action is used for creating a Vote - let's come back to the voting feature until some core functionality is built out. Ideally votes live on their own table and are associated with the user who created the vote.

_We would expect to be able to interact with a `create` action before being able to edit Monsters_